docs/wiki/quick_start.md

```diff
@@ -11,6 +11,43 @@
   pip3 install poetry pyinstaller;
   ```
 
+## Requirments for insystem installation
+
+- Ubuntu 18.04 or later, MacOS 10.9 or more
+- packages [python3, git, poetry, nginx, redis]
+
+- Install on ubuntu
```

Comment:
- the two points Install on ubuntu and MacOs should be consistent,either `Ubuntu, MacOs` or `Install on Ubuntu, Install on MacOs`
- Ubuntu should start with uppercase?